code/_onclick/item_attack.dm

```diff
@@ -107,6 +107,8 @@
 		switch(damtype)
 			if("brute")
 				M.apply_damage(power,BRUTE)
+				if(isanimal(M) && power > 10)
+					M.handle_blood_splatter(get_dir(user.loc, M.loc))
```

Comment:
typically blood stuff is done in /mob/type/apply_damage(), does this need to be here?